ls1intum/Artemis
# `Programming exercises`: Update C template to v1.36

### Checklist
#### General



#### Changes affecting Programming Exercises

### Motivation and Context


The C test framework was outdated and is now updated to the most recent version.

### Description
The new C template includes `.gitattributes` files and introduces a consistent formatting of python code. In addition, small bugs have been fixed.

### Steps for Testing

1. Create a C programming exercise
2. Check whether solution is graded with full points

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage
